tests/integration/test_sqs.py

```diff
@@ -437,29 +437,84 @@ def _send_message():
         ), f"unexpected number of messages in response {response}"
 
     @pytest.mark.aws_validated
-    @pytest.mark.xfail(reason="see https://github.com/localstack/localstack/issues/5938")
+    def test_create_queue_with_default_attributes_is_idempotent(self, sqs_client, sqs_create_queue):
+        queue_name = f"queue-{short_uid()}"
+        attributes = {
+            "VisibilityTimeout": "69",
```

Comment:
typo? because that seems to be a strange number to pick out of all available numbers ;)

Reply:
yes i make that mistake all the time. the keys are like right next to eachother